guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -174,32 +242,73 @@ Incremental upgrade (of a content view):: The act of creating a new (minor) cont
+endif::[]
+
 [[Job]]
 Job:: A command executed remotely on a host from {ProjectServer}.
 Every job is defined in a job template.
 
 [[Katello]]
-Katello:: A Foreman plugin responsible for subscription and repository management.
+Katello::
+Katello is an open source plugin to perform content management and subscription handling.
+It depends on Pulp for content management, which fetches software from repositories and stores various versions of it.
+It also depends on Candlepin for host registration and managing subscription manifests.
+ifdef::satellite,orcharhino[]
+The Katello plugin is always installed on {Project}.
+endif::[]
+
+[[Katello_Host_Tools_and_Katello_Host_Tools_Tracer]]
+Katello Host Tools and Katello Host Tools Tracer::
+The client-side software is responsible for the connection between hosts and content from {ProjectServer} or {SmartProxyServers}.
+It uploads currently used repositories and installed packages to {Project} and signals if the host needs to be rebooted after updating the Linux Kernel.
 
 [[Lazy_sync]]
 Lazy sync:: The ability to change the default download policy of a repository from *Immediate* to *On Demand*.
 The *On Demand* setting saves storage space and synchronization time by only downloading the packages when requested by a host.
 
 [[Location]]
-Location:: A collection of default settings that represent a physical place.
+Location::
+A collection of default settings that represent a physical place.
+Location is a tag mostly used for geographical separation of hosts within {Project}.
+Examples include different cities or different data centres.
 
 [[Library]]
 Library:: A container for content from all synchronized repositories on {ProjectServer}.
 Libraries exist by default for each organization as the root of every lifecycle environment path and the source of content for every content view.
 
 [[Lifecycle_environment]]
-Lifecycle environment:: A container for content view versions consumed by the content hosts.
-A Lifecycle Environment represents a step in the lifecycle environment path.
+Lifecycle environment::
+A lifecycle environment defines the stage in which certain versions of content are available to hosts and represents a step in the lifecycle environment path.
 Content moves through lifecycle environments by publishing and promoting content views.
+A typical scenario for lifecycle management is to distinguish between hosts for _development_, _testing_, and _production_.
+In this scenario, hosts are assigned to either the _development_, _testing_, or _production_ lifecycle environment and therefore run different versions of the same software in the different environments.
+This way, new versions of a software can be developed, and then tested before being used in the _production_ environment, greatly reducing the risk of disruption by prematurely rolled out updates.
```

Comment:
```suggestion
Lifecycle environment::
A lifecycle environment represents a step in the lifecycle environment path.
It defines the stage in which certain versions of content are available to hosts, such as _development_, _testing_, and _production_.
This way, new versions of software can be developed and tested before being deployed in a _production_ environment, thus reducing the risk of disruption by prematurely rolled out updates.
Content moves through lifecycle environments by publishing and promoting content views.
```
An attempt to simplify this. I think it's better to lead with the LE path.

Reply:
Thanks, applied!